lib-injection/README.md

```diff
@@ -0,0 +1,30 @@
+# Host injection
+
+Our gitlab build pipeline ships a pre-installed tracer deb/rpm package.
+
+Currently, we support
+
+| Environment| version |
+|---|---|
+| Ruby  | `2.7`, `3.0`, `3.1`, `3.2`|
+| OS    | Debian 10+ |
```

Comment:
Are we still building `.rpm`? Because Debian doesn't use `rpm`, so it looks like this list is out-of-sync with what we're actually generating?